src/ctelnet.cpp

```diff
@@ -814,7 +789,7 @@ void cTelnet::processTelnetCommand( const string & command )
           else if (!myOptionState[255])
           //only if the option is currently disabled
           {
-              if( //( option == OPT_SUPPRESS_GA ) ||
```

Comment:
Commenting this out/removing it also makes line 822 | 797 (before|after) unreachable...!